data/human/news.txt

```diff
@@ -2150,3 +2150,313 @@ news "graffiti on spaceport wall"
+
+phrase "Careless Ends Poorly"
+	word
+		"ignores air traffic control and heads to the bar. The ship"
+		"argues with the controller about their landing. The ship"
+		"turns around to reenter the ship, however it"
+	word
+		" is "
+	word
+		"struck "
+		"caught "
+	word
+		"by a wave and "
+	word
+		"swept into the sea."
+		"pushed off the pad."
+
+news "Careless Landing"
+	location
+		planet Jakobsen
+	name
+		"Recklessness"
+	message
+		phrase 
+			"A Light Ship"
+		word
+			" lands on a pad meant for heavier ships. Air traffic control is immediately upon the captain to move ${Pronoun} ship. The captain "
+		phrase
+			"Careless Ends Well"
+			"Careless Ends Poorly"
+
+phrase "CCOR Song Description"
+	word
+		"a song about "
+	word
+		"community, everyone pitching in where they can"
+		"the tyranny of a man name Bailey, until he was overthrown"
+		"CCOR, praising their accomplishments"
+		"resistance, opposition to the current planetary administration"
+
+phrase "Worker's Demands"
+	word
+		"safety regulations"
+		"increased pay"
+		"restoration of the CCOR"
+		"justice against planetary security brutality"
+		"immediate repositioning of the spaceport"
+
+news "Discontent"
+	location
+		planet Bailey
+	name
+		"Conflict"
+	message
+		word
+			"Allegations that planetary authorities will no longer move the spaceport has provoked "
+		word
+			"large scale "
+			"mass "
+			"moderate "
+			"scattered "
+		word
+			"riots "
+			"protests %"
+		word
+			"the rioters are "
+		word
+			"singing, %{CCOR Song Description}. "
+			"chanting, a list of their demands including ${Worker's Demands}. "
+			"sizing up the planetary security forces. "
+		word
+			"Security eventually steps in, "
+		word
+			"however they're pushed back. "
+			"but they're routed. "
+			"and they disperse the demonstrators. "
+			"and the demonstrators scatter but don't appear dissuaded."
+		replace
+			"% the rioters" "the protesters"
+			"%" ""
+
+news "Clandestine"
+	location
+		planet Alvorada
+name
+	"Observation"
+message
+	word
+		"You can't escape the feeling of being watched. It"
+	word
+		"'s probably the security cameras; most of them are still running."
+		" feels like there are eyes around every corner."
+		"'s the pirates; a small number of them seem to be observing you."
+	word
+		"After some time of this "
+	word
+		"a silent alarm tells you that your should check on your ship. "
+		"it may be wise make sure no-one's tampering with your ship. "
+	word
+		"There's "
+	word
+		"a lone pirate prodding the ship with rudimentary tools. "
+		"a few pirates watching but not apparently touching it. "
+		"noone othere, the area around your ship is almost suspiciously devioid of people. "
+	word
+		"Whatever's going on your ship appears unscathed."
+
+news "Microgravity"
+	location
+		planet Mordente-Bridi
+name
+	"Repairs"
+message
+	word
+		"Watching the micro-gravity section of the spaceport from the bar, you "
```

Comment:
So should it be microgravity or micro-gravity?

Reply:
I don't know. English is my first language but it's never made sense to me.